faq.adoc

```diff
@@ -28,6 +28,15 @@ So far Kaitai Struct focuses on reading (parsing) only. There are plans to suppo
 
 == How does it compare to ...
 
+=== ... Python library Construct?
+
+Actually, Construct is the closest analog to Kaitai Struct. It is also a symmetric binary parsing library, but the difference is mostly that:
+
+* Construct does building (serialization) and parsing while Kaitai ([currently](https://github.com/kaitai-io/kaitai_struct/issues/27)) only does parsing
```

Comment:
maybe
>does both

Reply:
And I guess that rewriting commits to keep the commit tree clean is fine if these commits are not cloned by anyone.

Reply:
Fixed and amended.